sg14-error-codes.html

```diff
@@ -325,7 +346,13 @@
         instead <code>libB</code> should expect its callers to write <code>if (LibB::is_oom_failure(code))</code>, where
         <code>bool LibB::is_oom_failure(std::error_code)</code> is a free function provided by <code>libB</code>. This successfully
         accomplishes semantic classification, and does it without any operator overloading, and therefore does it without the
-        need for the <code>std::error_condition</code> type.</li>
+        need for the <code>std::error_condition</code> type.
+      
+          <!-- You're forgetting one of the great strengths of the present error code design, it lets you tie together error codes from
+          multiple third party libraries without having to touch their source code. Via creating your own conditions, you can get the
+          "address not found" error code in third party library A, B and C to all interoperate. That was Beman's intent, and it works well -->
```

Comment:
I'm thinking of a C++ program which has dependencies on three third party C libraries, each with their own error coding. And the C++ program, rather than sprinkle switch statements everywhere, simply wraps up those C libraries into error codes and creates a set of common error conditions so the C++ code is not cluttered with third-party-library-specifics. This was Beman's original design intent, and it does work well for this.